tools/buildsteps/windows/x64-uwp/make-addons.bat

```diff
@@ -0,0 +1,8 @@
+@ECHO OFF
+
+rem set Visual C++ build environment
+call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat" amd64 store 10.0.14393.0
```

Comment:
I know, `x64`  is alternative for `amd64`. MS uses `amd64`, so I just follow MS style.